aspnetcore/fundamentals/minimal-apis/security.md

```diff
@@ -0,0 +1,197 @@
+
+```csharp
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddAuthentication().AddJwtBearer();
+builder.Services.AddAuthorization();
+
+var app = builder.Build();
+
+app.Run();
+```
+
+In some cases, such as controlling middleware order, it's necessary to explicitly register authentication and authorization. In the following sample, the authentication middleware runs _after_ the CORS middleware has run.
+
+```csharp
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddAuthentication().AddJwtBearer();
+builder.Services.AddAuthorization();
+
+var app = builder.Build();
+
+app.UseCors();
+app.UseAuthentication();
+app.UseAuthorization();
+
+app.Run();
+```
+
+### Configuring authentication strategy
+
+Authentication strategies typically support a variety of configurations that are loaded in via options. Minimal application support loading options from configuration for the following authentication strategies:
+
+- JWT bearer-based authentication strategies
+- OpenID Connection-based authentication strategies
+
+The ASP.NET framework expects to find these options under the `Authentication:Schemes:{SchemeName}` section in configuration. The `appsettings.json` definition below defines two different schemes, `Bearer` and `LocalAuthIssuer`, with their respective options. The `Authentication:DefaultScheme` option can be used to configure the default authentication strategy that will be used.
+
+```json
+{
+  "Authentication": {
+    "DefaultScheme":  "LocalAuthIssuer",
+    "Schemes": {
+      "Bearer": {
+        "ValidAudiences": [
+          "https://localhost:7259",
+          "http://localhost:5259"
+        ],
+        "ValidIssuer": "dotnet-user-jwts"
+      },
+      "LocalAuthIssuer": {
+        "ValidAudiences": [
+          "https://localhost:7259",
+          "http://localhost:5259"
+        ],
+        "ValidIssuer": "local-auth"
+      }
+    }
+  }
+}
+```
+
+In `Program.cs`, we register two JWT bearer-based authentication strategies: one with the default scheme name ("Bearer") and one with the "LocalAuthIssuer" scheme name. 
+
+> [!NOTE]
+> The scheme name is used to uniquely identify an authentication strategy and is used as the lookup key when resolving authentication options from config.
+
+```csharp
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddAuthentication()
+  .AddJwtBearer()
+  .AddJwtBearer("LocalAuthIssuer");
+  
+var app = builder.Build();
+
+app.Run();
+```
+
+## Configuring authorization policies in minimal applications
+
+While authentication is used to identify and validate the identity of users against an API, authorization is used to validate and verify access to resources in an API and is faciliated by the `IAuthorizationService` registered by the `AddAuthorization` extension method. In the scenario below, we want to add an `/hello` resource that requires a user to present an `is_admin` claim with a `greetings_api` scope.
```

Comment:
Must fix spelling facilitated if you reject my suggestion. Sentence is way to long for MT (machine translation).

```suggestion
Authentication is used to identify and validate the identity of users against an API. Authorization is used to validate and verify access to resources in an API and is facilitated by the `IAuthorizationService` registered by the `AddAuthorization` extension method. In the following scenario, a `/hello` resource is added that requires a user to present an `is_admin` claim with a `greetings_api` scope.
```